lib/shopify-cli/identity_auth.rb

```diff
@@ -237,13 +237,15 @@ def challange_params
     def auth_url
       if Environment.use_local_partners_instance?
         "https://identity.myshopify.io/oauth"
+      elsif Environment.use_spin_partners_instance?
+        "https://identity.#{Environment.spin_url}/oauth"
       else
         "https://accounts.shopify.com/oauth"
       end
     end
 
     def client_id_for_application(application_name)
-      client_ids = if Environment.use_local_partners_instance?
+      client_ids = if Environment.use_local_partners_instance? || Environment.use_spin_partners_instance?
```

Comment:
nit pick, but I wonder if there's a way you can just check if you're running in dev mode. Does `ctx.development?` work here?

Probably even better would be if the CLIENT_IDS were an env var, and there was no conditional logic here at all.

Reply:
@mkcny and I talked and this might be something we need to support with the installed version of the CLI. We can simplify the logic as we remove the stubbed and local configurations for scripts and solely rely on spin instances.